requirements/edx/testing.txt

```diff
@@ -737,7 +737,7 @@ loremipsum==1.0.5
     # via
     #   -r requirements/edx/base.txt
     #   ora2
-lti-consumer-xblock==2.11.0
+git+https://github.com/open-craft/xblock-lti-consumer.git@kshitij/bb-3899/pii-flag-cleanup#egg=lti-consumer-xblock==3.0.0
```

Comment:
Are we planning on keeping this or will we have an updated version of the `xblock-lti-consumer` available to be used instead?